modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -69,6 +82,34 @@ public AspNetCoreServerCodegen() {
         cliOptions.clear();
 
         // CLI options
+        addOption(CodegenConstants.LICENSE_URL,
+                CodegenConstants.LICENSE_URL_DESC,
+                licenseUrl);
+
+        addOption(CodegenConstants.LICENSE_NAME,
+                CodegenConstants.LICENSE_NAME_DESC,
+                licenseName);
+
+        addOption(CodegenConstants.PACKAGE_COPYRIGHT,
+                CodegenConstants.PACKAGE_COPYRIGHT_DESC,
+                packageCopyright);
+
+        addOption(CodegenConstants.PACKAGE_AUTHORS,
+                CodegenConstants.PACKAGE_AUTHORS_DESC,
+                packageAuthors);
+
+        addOption(CodegenConstants.PACKAGE_TITLE,
+                CodegenConstants.PACKAGE_TITLE_DESC,
+                packageTitle);
+
+        addOption(CodegenConstants.ARTIFACT_VERSION,
+                CodegenConstants.ARTIFACT_VERSION_DESC,
```

Comment:
Suggestion: Sorry for the adjacent comment, but it's the only way to make multi-line suggestions.

This description refers to something about pom.xml, and is intended to be used only for JVM based builds apparently.

I recommend wording similar to other `PACKAGE*` constants:
```suggestion
                "Specifies an assembly version for the .NET resource.",
```

Reply:
I realized I did not actually use artifactVerson finally so just remove it. packageVersion was already there.